index.html

```diff
@@ -7,14 +7,20 @@
 	<title>Portfolio</title>
 	<script src="https://kit.fontawesome.com/b8019b4db4.js" crossorigin="anonymous"></script>
 	<link rel="stylesheet" type="text/css" href="styles.css">
+
 	
 </head>
 
 <body>
 	<header class="container">
 		<div id="logo">Lassana</div>
-		<nav>
+		<nav class="nae toggle-menu">
 			<button id="hide" type="button"><i class="fas fa-bars"></i></button>
```

Comment:
As of the project requirement after the user clicks the `hamburger button` it should have to be changed to the close `(X)` button and should be displayed at the right top of the mobile menu screen.

| Your Design | Figma Design |
|-----------|--------------|
| <img width="313" alt="image" src="https://user-images.githubusercontent.com/34578631/162194509-be7afae2-c4d7-484d-96bc-4cc4e900972d.png"> | <img width="351" alt="image" src="https://user-images.githubusercontent.com/34578631/162194604-735ab3e0-3731-4d8c-a794-f98615bb514f.png"> |

Kindly, fix this too.